app-editors/remarkable/remarkable-1.9.0_pre20210320.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..10} )
+inherit desktop gnome2-utils python-single-r1
+
+DESCRIPTION="a fully featured markdown editor, supports github markdown dialect"
+HOMEPAGE="http://remarkableapp.github.io/ https://github.com/jamiemcg/remarkable"
+GIT_COMMIT="7b0b3dacef270a00c28e8852a88d74f72a3544d7"
+SRC_URI="https://github.com/jamiemcg/remarkable/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/Remarkable-${GIT_COMMIT}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+#IUSE="spellcheck"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	${PYTHON_DEPS}
+	net-libs/webkit-gtk
+	$(python_gen_cond_dep '
+		dev-python/beautifulsoup4[${PYTHON_USEDEP}]
+		dev-python/pycairo[${PYTHON_USEDEP}]
+		dev-python/pygobject[${PYTHON_USEDEP}]
+		dev-python/markdown[${PYTHON_USEDEP}]
+	')"
+RDEPEND="${DEPEND}"
+BDEPEND=""
+PATCHES=( "${FILESDIR}"/disable-spellcheck.patch )
```

Comment:
Why?

Reply:
Why what? please ask more specific

Reply:
Why include a disable spellcheck patch?

Reply:
I suppose looking further it may be because that package doesn't exist. If you wanted to you could add it here! Up to you of course

Reply:
Yes, that's the reason. pygtkspellcheck has no gentoo package, and I am not motivated to do this yet.